app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+    <string name="placeholder_session_language">ja</string>
+    <string name="placeholder_session_day">今日</string>
+    <string name="placeholder_session_start_time">17:15</string>
+    <string name="placeholder_session_duration">240分</string>
+    <string name="placeholder_session_room">Village:Königlich Bayerisches Amtsvillage</string>
+    <string name="placeholder_schedule_time">09:15</string>
+    <string name="placeholder_alarm_time">2013年12月28日　午後5時10分</string>
+    <string name="placeholder_session_date">2013年12月28日　午後5時15分</string>
+    <string name="placeholder_session_location">本当に広い部屋</string>
+    <string name="placeholder_session_id">ID: 5193</string>
+    <string name="placeholder_day_separator_title">2日目</string>
+    <string name="preference_title_auto_update_enabled">自動更新</string>
+    <string name="preference_dialog_title_alarm_time">デフォルトのアラーム時間を選択します</string>
+    <string name="preference_title_alarm_time">デフォルトのアラーム時間</string>
+    <string name="preference_title_alarm_tone">アラーム音</string>
+    <string name="preference_title_alternative_highlighting_enabled">代替の強調表示</string>
+    <string name="preference_title_alternative_schedule_url">代替のスケジュールURL</string>
+    <string name="preference_title_app_notification_settings">通知をカスタマイズ</string>
+    <string name="preference_title_insistent_alarms_enabled">しつこいアラーム</string>
+    <string name="preference_summary_engelsystem_json_export_url">"         <![CDATA[シフトを確認するには、ここに<u>個人</ u>のURLを入力してください。<br/><br/>         1. あなたのEngelsystemアカウントにログインします。<br/>         2.プロフィールページの<u> JSON出力</ u>ボタンの後ろにあるURLをコピーします.<br/>         3. URLをここに貼ります。<br/><br/>         メイン会議日の前後のシフトは<u>表示されません</ u>。]]>     "</string>
+    <string name="menu_item_title_about">情報</string>
+    <string name="menu_item_title_add_to_calendar">カレンダーに追加</string>
+    <string name="menu_item_title_alarms">アラーム</string>
+    <string name="menu_item_title_close_session_details">イベント詳細を閉じる</string>
+    <string name="menu_item_title_flag_as_favorite">お気に入りに追加</string>
+    <string name="menu_item_title_unflag_as_favorite">お気に入りから削除</string>
+    <string name="menu_item_title_delete_favorite">削除</string>
+    <string name="menu_item_title_set_alarm">アラーム設定</string>
+    <string name="menu_item_title_delete_alarm">アラーム削除</string>
+    <string name="menu_item_title_delete_all">全て削除</string>
+    <string name="menu_item_title_favorites">お気に入り</string>
+    <string name="menu_item_title_feedback">フィードバック</string>
+    <string name="menu_item_title_navigate">ナビゲート</string>
+    <string name="menu_item_title_refresh">リフレッシュ</string>
+    <string name="menu_item_title_share_session">イベントを共有</string>
+    <string name="menu_item_title_share_favorites">お気に入りを共有</string>
+    <string name="OK">OK</string>
+    <string name="certificate_error_message">"         安全な接続の確立に失敗しました。          (<xliff:g example="Unknown Error" id="error">%s</xliff:g>)     "</string>
```

Comment:
Please remove the surrounding quotes and whitespace. It might have been added by the editor.

Reply:
Thanks for removing the quotes. The whitespace can be removed, too.